.github/workflows/nightly_run.yml

```diff
@@ -599,6 +599,11 @@ jobs:
         run: bash build.sh --sysroot "$SYSROOT_PATH"
         working-directory: ./core/iwasm/libraries/lib-wasi-threads/test/
 
+      - name: Build WASI thread stress tests
```

Comment:
why do we do it on both nightly and on every change? Isn't the test on every change sufficient?

Reply:
we can run them only in nightly, I don't think it changes much, but we can also test per-pr as it's 10-15 min 

I think both are ok, maybe let's disable them to make per-pr CI faster

Reply:
I think we can keep them per-pr as you have now (at least for now, until they start taking longer), but if it's per-pr, I don't see much value having it in nightly as well.